torch/csrc/autograd/python_function.cpp

```diff
@@ -507,6 +507,13 @@ static void _wrap_outputs(
     return results;
   };
 
+  auto view_as_fn = [](const at::Tensor& x, const at::Tensor& y) -> at::Tensor {
+    pybind11::gil_scoped_acquire gil;
```

Comment:
I'm not actually sure we need this since we are calling from Python and we don't release the gil, but jvp_user_function does it.